ComponentKit/Core/CKComponentViewConfiguration.mm

```diff
@@ -20,7 +20,11 @@
 
 CKComponentViewClass::CKComponentViewClass(Class viewClass) noexcept :
 identifier(class_getName(viewClass)),
-factory(^{return [[viewClass alloc] init];}) {}
+factory(^{ return [[viewClass alloc] init]; }) {
+  #if DEBUG
```

Comment:
Should we remove it from https://github.com/facebook/componentkit/blob/ed357f67df3d398cccc536a0c7b280e3e0b62ea8/ComponentKit/Core/CKComponentViewConfiguration.mm#L59, too?